andela/ah-bird-box-frontend
# #163388215 Users will Fetch Single Article, Update and Delete the article

#### What does this PR do?
This PR will enable a user to view a single article, update the article if it is theirs and delete the article

#### Description of Task to be completed?
- User to fetch a single article
- User to update an article
- User to delete the article
- Improves the navigation bar for the app
- Introduces loaders to the application

#### How should this be manually tested?
(_Note: Screenshots below indicate valid urls(add `s` to `article`) and slug which can be used for test_)
__Fetch Single Article__
Follow the address `/articles/:slug` to view the article with the specified slug

__Update Article__
From the above address, `/articles/:slug` click the edit icon on the right of the article, the icon appears only if you own the article

__Delete Article__
From the same address, `/article/:slug`, find the delete icon at the right of the page

#### What are the relevant pivotal tracker stories?
[#163388215](https://www.pivotaltracker.com/story/show/163388215)

#### Screenshots (if appropriate):
_1. Article Edit Page_
![image](https://user-images.githubusercontent.com/29451552/54786076-e7105e80-4c38-11e9-8c4e-f20eef31e157.png)

_2. Single Article page with update and delete controls_
![image](https://user-images.githubusercontent.com/29451552/54786043-d5c75200-4c38-11e9-978d-dc4069127a01.png)

#### Reviewers:
@TooColline
@calebrotich10
@unah254
@bryan-munene
@hoslack
